core/services/fluxmonitor/flux_monitor.go

```diff
@@ -348,20 +350,22 @@ func NewPollingDeviationChecker(
 	pollDelay time.Duration,
 ) (*PollingDeviationChecker, error) {
 	return &PollingDeviationChecker{
-		store:          store,
-		fluxAggregator: fluxAggregator,
-		initr:          initr,
-		requestData:    initr.InitiatorParams.RequestData,
-		idleThreshold:  initr.InitiatorParams.IdleThreshold.Duration(),
-		threshold:      float64(initr.InitiatorParams.Threshold),
-		precision:      initr.InitiatorParams.Precision,
-		runManager:     runManager,
-		fetcher:        fetcher,
-		pollTicker:     NewResettableTicker(pollDelay),
-		idleTicker:     nil,
-		chMaybeLogs:    make(chan maybeLog, 100),
-		chStop:         make(chan struct{}),
-		waitOnStop:     make(chan struct{}),
+		store:              store,
+		fluxAggregator:     fluxAggregator,
+		initr:              initr,
+		requestData:        initr.InitiatorParams.RequestData,
+		idleThreshold:      initr.InitiatorParams.IdleThreshold.Duration(),
+		threshold:          float64(initr.InitiatorParams.Threshold),
+		precision:          initr.InitiatorParams.Precision,
+		runManager:         runManager,
+		fetcher:            fetcher,
+		pollTicker:         NewResettableTicker(pollDelay),
+		idleTicker:         nil,
+		roundTimeoutTicker: nil,
+		roundTimeout:       0,
```

Comment:
roundTimeout looks unused

Reply:
agreed. looks like we just pull the timeout from the contract?